### DISCUSSION TITLE
O que preciso para instalação do Angular

### DISCUSSION THREAD

**Discussion by scooweb** [po-ui/po-angular#1546] [Perguntas frequentes]
Olá gostaria de saber como faço para instalar o Angular?

**Comment by brunoromeiro** [comment]
Olá @scooweb tudo bem?

Para fazer a instalação do Angular precisa seguir esses passos aqui da [documentação oficial do Angular](https://angular.io/guide/setup-local), ou seguir a [nossa própria documentação](https://po-ui.io/guides/getting-started).

Em ambas você precisa ter o [NodeJS](https://nodejs.org/pt-br/) instalado.
